modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -25,14 +25,22 @@ To integrate the built-in {ocp} OAuth server as an identity provider for {produc
 For security, Red Hat recommends first setting the *Minimum access role* to *None* while you complete setup. Later, you can return to the *Access Control* page to set up more tailored access rules based on user metadata from your identity provider.
 ====
 
-. To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section. For example:
-.. To give the *Admin* role to a user called `administrator`, you can use the following key-value pairs to create access rules:
+. To add access rules for users and groups accessing {product-title-short}, click *Add new rule* in the *Rules* section. You will need attributes for the user to configure their access. To get user information in {ocp}, click *User Management* -> *Users* -> *<username*> -> YAML or go to `k8s/cluster/user.openshift.io-v1-User/<username>/yaml` and note the values for `name`, `uid` (`userid` in {product-title-short}), and `groups`. 
++
+.. For example, to give the *Admin* role to a user named `administrator` with the following attributes:
+
+- `uid`: `12345-00aa-1234-123b-123fcdef1234`
+-  `groups`: `["system:authenticated", "system:authenticated:oauth"]`
++
+You can use the following key-value pairs to create an access rule based on the `name` (Row 1) or `uid`/`userid` (Row 2):
 +
 |===
-| *Key* | *Value*
+| *Key* | *Value* | *Role*
 |Name
 |administrator
-|Role
+|Admin
+|userid
+|12345-00aa-1234-123b-123fcdef1234
 |Admin
```

Comment:
```suggestion
|Admin
|groups
|myAdministratorsGroup
|Admin
```

Let's also say that using group mappings (rules) is recommended because it's more robust. Groups are usually associated with teams or permission sets and hence are more stable and hence rules require modifications less often.